ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -13,6 +13,12 @@ import BasicCardBox from '@akashaorg/design-system-core/lib/components/BasicCard
 import { useGetLogin, useLogout } from '@akashaorg/ui-awf-hooks';
 
 const SidebarComponent: React.FC<RootComponentProps> = props => {
+  const [isMobile, setIsMobile] = React.useState(window.matchMedia('(max-width: 1439px)').matches);
+
+  React.useEffect(() => {
+    setIsMobile(window.matchMedia('(max-width: 1439px)').matches);
+  }, [window.matchMedia('(max-width: 1439px)').matches]);
```

Comment:
can we have these breakpoints (1440, 1439 etc) defined somewhere and imported as variables instead? This makes it easier to update/maintain in future

Reply:
I've defined them somewhere.